pdr_backend/ppss/abstract/base_ss.py

```diff
@@ -0,0 +1,158 @@
+from abc import ABC
+from typing import Dict, List, Optional, Set, Tuple, Union
+
+from enforce_typing import enforce_types
+
+from pdr_backend.cli.arg_feed import ArgFeed, ArgFeeds
+from pdr_backend.cli.arg_pair import ArgPair
+from pdr_backend.subgraph.subgraph_feed import SubgraphFeed
+
+
+class MultiFeedSS(ABC):
```

Comment:
I'd prefer if this class was labeled as, and treated more like a mixin. Where other ss classes mix in a slice of info about feeds. Therefore it does not need to be abstract.

Same thing fir the single feed class below.